docs/event-service/3_getting-started.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 3
+---
+
+# Getting Started
+
+Before setting up the event service, ensure you have the following prerequisites
+
+### Prerequisites
+
+- [Node.js] - Version 20 or later
+- [NestJS] - Familiarity with NestJS framework
+- [PostgreSQL] - Database for storing event data
+
+[//]: #
+[Node.js]: http://nodejs.org
+[NestJS]: https://docs.nestjs.com
+[PostgreSQL]: https://www.postgresql.org/
```

Comment:
Also, add instructions about how to set this up locally? Only requisites is not enough?